src/main/java/fr/xephi/authme/process/join/AsynchronousJoin.java

```diff
@@ -191,6 +195,15 @@ private void processJoinSync(Player player, boolean isAuthAvailable) {
                 int blindTimeOut = (registrationTimeout <= 0) ? 99999 : registrationTimeout;
                 player.addPotionEffect(new PotionEffect(PotionEffectType.BLINDNESS, blindTimeOut, 2));
             }
+
+            if (service.getProperty(RestrictionSettings.SPECTATE_STAND_LOGIN)) {
+                // The delay is necessary in order to make sure that the player is teleported to spawn
```

Comment:
What about listening at the player spawn/respawn event?

Reply:
\+ doesn't the armor stand have to be deleted if the player disconnects as well?

Reply:
> What about listening at the player spawn/respawn event?

Looks like this will also require a delay, because at this stage it's impossible to change the player gamemode

Reply:
> + doesn't the armor stand have to be deleted if the player disconnects as well?

Armorstands are removed if an unauthorized player leaves the server